docs/open_source/releases-and-branches.md

```diff
@@ -0,0 +1,73 @@
+# Releases and Branches
+
+This document describes the branching and release strategy we use for MDC Web, and how it affects dependent repositories
+we maintain, namely the [MDC Web Catalog](https://github.com/material-components/material-components-web-catalog) and
+[MDC React](https://github.com/material-components/material-components-web-react).
+
+For an explanation of the steps needed to cut a release of MDC Web, see [Release Process](./release-process.md).
+
+## MDC Web Release Schedule and Versioning
+
+MDC Web follows a 2-week release cycle. We expect to have one release per month containing breaking changes, to coincide
+with the [Material Design Roadmap](https://github.com/material-components/material-components/blob/develop/ROADMAP.md).
+Any other interim releases will be patch releases including any non-breaking / non-feature changes that can be
+cherry-picked from master in a straightforward manner.
+
+This is a change from our earlier process, where we simply tagged a release with whatever has been merged to master
+within the two-week cycle, and would rev the version number accordingly based on whether there were breaking changes.
+
+## Release Types
+
+### Breaking-change Releases
+
+These are currently our minor releases, e.g. 0.35.0, 0.36.0, etc. These follow the currently-outlined
+[Release Process](./release-process.md) and are performed against `master`.
+
+### Pre-releases
+
+MDC Web Catalog and MDC React both depend on MDC Web via npm dependencies. In order to be able to integrate
+breaking changes for the upcoming release ahead of time - preferably while the necessary changes are fresh in someone's
+mind - we will cut pre-releases of MDC Web when breaking changes are merged which warrant updates in the other
+repositories. These will be cut on an as-needed basis, with no set schedule.
+
+The Catalog and React repositories will each have a `next` branch, which will be updated to point to the pre-release.
+Pull requests should be filed against this branch for changes needed for the upcoming MDC Web release.
+
+Later, when the new version of MDC Web is released, the `next` branch will again be updated to point to the new release,
+and we will open a PR to squash & merge its contents onto `master`. Once this is done, the `next` branch will be reset
+against `master` to accommodate the subsequent release.
+
+The release process for pre-releases should be largely similar to for regular releases, with minor changes
```

Comment:
"should be largely similar to regular releases"

Reply:
Oops. I accidentally some words in this sentence.